### PR TITLE
Masking Tomcat Version

### PR DESCRIPTION
**Pull request by DukeDavis12**
Masking Tomcat Version to reduce information disclosure.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

**Review by shrikant1407 — APPROVED** [review]

**Review by v-vasavi — COMMENTED** [review]
Is there a blanket setting to turn all versions off?

**Comment by DukeDavis12** [comment]
> Is there a blanket setting to turn all versions off?

`showReport="false" showServerInfo="false"` property 
ensures that Tomcat-related version or error report is not displayed.

File: pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
     <slf4j.version>1.7.36</slf4j.version>
     <mariadb.version>3.0.5</mariadb.version>
     <hibernate.version>6.1.0.Final</hibernate.version>
-    <tomcat.version>10.1.0-M16</tomcat.version>
+    <tomcat.version>10.1.1</tomcat.version>
```
Comment (tajnisha) [inline pom.xml:46]: Any reason for upgrade ?

File: component-samples/demo/aio/context.xml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Context>
   <Realm className="org.apache.catalina.realm.MemoryRealm" pathname="tomcat-users.xml"/>
+  <Valve className="org.apache.catalina.valves.ErrorReportValve" showReport="false" showServerInfo="false"/>
```
Comment (tajnisha) [inline component-samples/demo/aio/context.xml:4]: Masking required for for both debug and release traces ?

File: pom.xml (hunk 1 above)
Comment (DukeDavis12) [reply pom.xml:46]: Tomcat 10.1.0-* was flagged for vulnerabilities in BDBA

File: component-samples/demo/aio/context.xml (hunk 2 above)
Comment (DukeDavis12) [reply component-samples/demo/aio/context.xml:4]: Yes

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**